src/main/scala/com/fulcrumgenomics/sv/tools/SvPileup.scala

```diff
@@ -95,8 +94,8 @@ class SvPileup
   /** Coalesce the breakpoint counts and write them. */
   private def writeBreakpoints(tracker: BreakpointTracker, dict: SequenceDictionary): Unit = {
     val writer = Io.toWriter(output)
-    val fields = Seq("id", "left_contig", "left_pos", "left_strand", "right_contig", "right_pos", "right_strand", "total") ++
-      EvidenceType.values.map(_.snakeName)
+    val fields = Seq("id", "left_contig", "left_pos", "left_strand", "right_contig", "right_pos", "right_strand",
+      "split_reads", "read_pairs", "total")
```

Comment:
Now that we have a fixed (2) set of evidence types, we could convert this to a `Metric` which would make reading it back and working on it later much easier.